pom.xml

```diff
@@ -17,11 +17,11 @@
 
 	<properties>
 		<java.version>17</java.version>
-		<spring-cloud.version>2023.0.0</spring-cloud.version>
+		<spring-cloud.version>2023.0.2</spring-cloud.version>
 		<snakeyaml.version>2.2</snakeyaml.version>
-		<swagger-annotations.version>2.2.20</swagger-annotations.version>
+		<swagger-annotations.version>2.2.22</swagger-annotations.version>
 		<modelmapper.version>3.2.0</modelmapper.version>
-		<kafka.version>3.6.1</kafka.version>
+		<kafka.version>3.7.0</kafka.version>
```

Comment:
i suggest to remove this version because it is already included in spring boot 3.3.0